app/assets/stylesheets/custom.css.scss

```diff
@@ -1,5 +1,6 @@
 $baseFontSize:          13px !default;
 $baseLineHeight:        18px !default;
+$sampleAnswerBorder:    #ddd;
```

Comment:
Name of variable `sampleAnswerBorder` should be written in all lowercase letters with hyphens instead of underscores